modules/trusted-firmware-m/tfm_boards/common/nrf_provisioning.c

```diff
@@ -16,6 +16,51 @@
 #include "nrf_provisioning.h"
 #include <identity_key.h>
 #include <tfm_spm_log.h>
+#ifdef NRF53_SERIES
+#include <dfu/pcd_common.h>
+#include <spu.h>
+#include <hal/nrf_reset.h>
+
+static struct pcd_cmd *cmd = (struct pcd_cmd *)PCD_CMD_ADDRESS;
+
+static void pcd_write_cmd_lock_debug(void)
+{
+	cmd->magic = PCD_CMD_MAGIC_LOCK_DEBUG;
+	cmd->data = NULL;
+	cmd->len = 0;
+	cmd->offset = 0;
```

Comment:
```suggestion
	*cmd = (struct pcd_cmd) {
		.magic = PCD_CMD_MAGIC_LOCK_DEBUG,
	};
```
Doesn't it look nicer when you just fill in the values you need and let compiler to zero pad rest of the structure. This way you don't accidentally leave unfilled members if this struct is going to be extended.

Reply:
Thanks! Excellent reminder on how this should be done.